README.md

```diff
@@ -7,10 +7,9 @@
 
 Searvey aims to provide the following functionality:
 
-- Unified catalogue of observational data including near real time.
+- Unified catalogue of observational data including near real time (WIP).
 
-- Real time data analysis/clean up to facilitate comparison with numerical
-  models.
```

Comment:
Since you're already modifying this file, can you please also separate the line for VLIZ from IOC? I missed it when I added USGS

Reply:
sure